client/src/sections/courses/create/InitialSection.tsx

```diff
@@ -25,15 +26,21 @@ interface InitialSectionProps {
 const InitialSection: FC<InitialSectionProps> = ({ register, errors, control }) => {
   const { values: course, editMode } = useAppSelector((state) => state.course);
   const dispatch = useAppDispatch();
-  const categoriesOption: MultiSelectOptionData[] = categories.map(({ id, name }) => ({
-    value: id,
-    label: name,
-  }));
 
   useEffect(() => {
     dispatch(changeEditMode(true));
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
+
+  const {
+    data: categoryData,
+  } = useGetCategoryQuery(undefined);
```

Comment:
because I use builder.query and it is not needed a parameter same as getCourses 
![image](https://github.com/framgia/sph-lms/assets/110363969/3a3806a6-0991-4b0b-94fb-8e84766f434f)
![image](https://github.com/framgia/sph-lms/assets/110363969/a498e05d-2cc3-4e85-960d-ccdd205a936a)

Reply:
@FranzDelosSantos if its not expecting a parameter, why we are passing `undefinied` ?

Reply:
because an error in my end will occcur saying Expected 1-2 arguments, but got 0.

![image](https://github.com/framgia/sph-lms/assets/110363969/5b6264f3-00aa-41a2-a7e4-cda879566804)

Reply:
That is because, You have to pass type in the query